scripts/appleseedMaya/AETemplates/NEasBumpTemplate.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<templates>
+    <using package='maya'/>
+    <template name='NEasBump'>
+        <attribute name='outNormal' type='maya.float3'>
+            <label>Out Normal</label>
+        </attribute>
+        <attribute name='bumpValue' type='maya.float'>
+            <label>Bump Value</label>
+        </attribute>
+        <attribute name='bumpDepth' type='maya.float'>
+            <label>Bump Depth</label>
+        </attribute>
+        <attribute name='asNormalMap' type='maya.float3'>
+            <label>As Normal Map</label>
```

Comment:
What does "As" mean here? Is it the English word "as", or does it mean appleseed?

Reply:
Appleseed, but Maya capitalizes it automagically. I removed the *as* prefix. Sometimes this is added to avoid conflict with some of Maya's built-in nodes, but it's not necessary in this case.